cordova-lib/src/cordova/prepare.js

```diff
@@ -37,23 +37,25 @@ function prepare(options) {
     return Q().then(function() {
         var projectRoot = cordova_util.cdProjectRoot();
         var config_json = config.read(projectRoot);
-        options = options || { verbose: false, platforms: [], options: {} };
+        options = options || { verbose: false, options: {} };
+        options.platforms = options.platforms || cordova_util.listPlatforms(projectRoot);
```

Comment:
No, because preProcessOptions will throw `No platforms added to this project` on restoring ([save.spec](https://github.com/apache/cordova-lib/blob/2a178662acfad9c779114fddd815e484fc1b6f1d/cordova-lib/spec-cordova/save.spec.js#L613)).